minichain/mempool.py

```
@@ -17,62 +16,62 @@ def add_transaction(self, tx):
+                if tx.timestamp <= existing_tx.timestamp:
                     logger.warning("Mempool: Ignoring older replacement %s", tx.tx_id)
                     return False
                 
+                self._list.pop(existing_idx)
+                if i_max > existing_idx:
+                    i_max -= 1
+                if i_min > existing_idx:
+                    i_min -= 1
             else:
-                if self._size >= self.max_size:
+                if len(self._list) >= self.max_size:
```

Ruff (0.15.15)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)